{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Product defects","children":[{"name":"selene.core.exceptions.TimeoutException: Message: \n\nTimed out after 4s, while waiting for:\nbrowser.element(('css selector', '.header-search-button')).click\n\nReason: ElementNotInteractableException: Message: element not interactable\n  (Session info: headless chrome=114.0.5735.198)\nStacktrace:\n#0 0x55ce974c84e3 <unknown>\n#1 0x55ce971f7b00 <unknown>\n#2 0x55ce97234fa1 <unknown>\n#3 0x55ce9722a1da <unknown>\n#4 0x55ce97253012 <unknown>\n#5 0x55ce97229b03 <unknown>\n#6 0x55ce972531de <unknown>\n#7 0x55ce9726b30e <unknown>\n#8 0x55ce97252de3 <unknown>\n#9 0x55ce972282dd <unknown>\n#10 0x55ce9722934e <unknown>\n#11 0x55ce974883e4 <unknown>\n#12 0x55ce9748c3d7 <unknown>\n#13 0x55ce97496b20 <unknown>\n#14 0x55ce9748d023 <unknown>\n#15 0x55ce9745b1aa <unknown>\n#16 0x55ce974b16b8 <unknown>\n#17 0x55ce974b1847 <unknown>\n#18 0x55ce974c1243 <unknown>\n#19 0x7f394dd43b43 <unknown>\n\nScreenshot: file:///home/runner/.selene/screenshots/1688895562420/1688895562420.png\nPageSource: file:///home/runner/.selene/screenshots/1688895562420/1688895562420.html","children":[{"name":"test_decorator_steps","uid":"3a48beeb078deec4","parentUid":"8eb7d3949671c59029f1db08c47311e7","status":"failed","time":{"start":1688895564266,"stop":1688895569933,"duration":5667},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"8eb7d3949671c59029f1db08c47311e7"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"},{"name":"Test defects","children":[{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=54101): Max retries exceeded with url: /session/ae9518916230a560426e366287142165/window/maximize (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9a1633d050>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"test_github","uid":"3b2fdcc37ef37a48","parentUid":"6aa9b6982e9ee7d710ba8e06c0a4ce2e","status":"broken","time":{"start":1688895570082,"stop":1688895570082,"duration":0},"flaky":true,"newFailed":false,"newPassed":false,"newBroken":true,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"6aa9b6982e9ee7d710ba8e06c0a4ce2e"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=54101): Max retries exceeded with url: /session/ae9518916230a560426e366287142165/window/maximize (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9a14be0790>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"test_dynamic_steps","uid":"d1b1aef2fcdaad57","parentUid":"6fc01cce8da2a1c3ab92468cc49f4772","status":"broken","time":{"start":1688895570274,"stop":1688895570274,"duration":0},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":true,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"6fc01cce8da2a1c3ab92468cc49f4772"}],"uid":"bdbf199525818fae7a8651db9eafe741"}],"name":"categories"}